lib/mbta.js

```diff
@@ -1,9 +1,61 @@
-'use strict'
-
+"use strict"
 // Code here.
+const subwayLines = {
+  Red: ["South Station", "Park Street", "Kendall", "Central", "Harvard",
+    "Porter", "Davis", "Alewife"],
+  Green: ["Government Center", "Park Street", "Boylston", "Arlington", "Copley",
+    "Hynes", "Kenmore"],
+  Orange: ["North Station", "Haymarket", "Park Street", "State",
+    "Downtown Crossing", "Chinatown", "Back Bay", "Forest Hills"]
+}
 
-const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+const stopsBetweenStations = function (startLine, startStation, endLine, endStation) {
+  console.log("StartLine: "+ startLine + " StartStation: "+startStation)
+  console.log("EndLine: "+ endLine + " EhdStation: "+endStation)
+  let totalStops = "99"
+  // First check if StartLine and end line are the same.
+  if(startLine === endLine) {
+     console.log("Startline and EndLine are the same")
+     // array of stops
+     let startStops = subwayLines[startLine];
+     let startingIndex  = startStops.indexOf(startStation);
+     let endingIndex = startStops.indexOf(endStation);
 
+     console.log("startStops beginning at: "+startingIndex)
+     console.log("endStops at: "+endingIndex)
+     return "Number of stops: " + (endingIndex - startingIndex)
```

Comment:
The key here is to use `Math.abs` rather than simply subtracting the end index from the start index, in case the user is moving in the opposite direction